core/src/main/resources/hudson/tasks/ArtifactArchiver/help-flattenDirectories.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  Optionally flatten directories to be archived, so that all archived files are
```

Comment:
Battling Intellij's indent styles. I'll edit it outside of Intellij to get that right.